### PR TITLE
Sorting by `created_at` field for multiple queries

### PR DESCRIPTION
**Pull request by zedzior**
I want to merge this change, because I want to enable sorting by `created_at` field for all the queries, which are ran behind dashboard's tables. This PR adds the sorting to `giftCards` and `pages`.

Issue: https://github.com/saleor/saleor/issues/10907

# Impact

* [x] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/schema.graphql [hunk 1]
```diff
@@ -9939,6 +9943,13 @@ enum PageSortField {
   DEPRECATED: this field will be removed in Saleor 4.0.
   """
   PUBLISHED_AT
+
+  """
+  Sort pages by creation date.
+  
+  DEPRECATED: this field will be removed in Saleor 4.0.
```
Comment (zedzior) [inline saleor/graphql/schema.graphql:9950]: `npm run build-schema` command created wrong schema: it should say `Added in Saleor 3.8.` 
```
if self.name == "CREATION_DATE":
    description += DEPRECATED_IN_3X_INPUT
if self.name == "CREATED_AT":
    description += ADDED_IN_38
```
Same issue above for `PUBLISHED_AT`. It looks like, whenever a sorter is exactly the same, but different named, the problem occurs. Is it a known drawback of the `npm run build-schema` command?

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/giftcard/sorters.py [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
 
     @property
     def description(self):
         if self.name in GiftCardSortField.__enum__._member_names_:
             sort_name = self.name.lower().replace("_", " ")
-            return f"Sort orders by {sort_name}."
+            description = f"Sort gift cards by {sort_name}."
+            if self.name == "CREATED_AT":
+                description += ADDED_IN_38
+            return description
```
Comment (NyanKiyoshi) [inline saleor/graphql/giftcard/sorters.py:20]: There is no index:
https://github.com/saleor/saleor/blob/c28b12323141208257338c3619a57bd2dc8b1922/saleor/giftcard/models.py#L70

File: saleor/graphql/page/sorters.py [hunk 3]
```diff
@@ -19,6 +20,10 @@ def description(self):
             description = f"Sort pages by {sort_name}."
             if self.name == "PUBLICATION_DATE":
                 description += DEPRECATED_IN_3X_INPUT
+            if self.name == "CREATION_DATE":
+                description += DEPRECATED_IN_3X_INPUT
+            if self.name == "CREATED_AT":
+                description += ADDED_IN_38
```
Comment (NyanKiyoshi) [inline saleor/graphql/page/sorters.py:26]: There is no index:
https://github.com/saleor/saleor/blob/c28b12323141208257338c3619a57bd2dc8b1922/saleor/page/models.py#L26

File: saleor/graphql/page/sorters.py [hunk 4]
```diff
@@ -19,6 +20,10 @@ def description(self):
             description = f"Sort pages by {sort_name}."
             if self.name == "PUBLICATION_DATE":
                 description += DEPRECATED_IN_3X_INPUT
+            if self.name == "CREATION_DATE":
+                description += DEPRECATED_IN_3X_INPUT
```
Comment (NyanKiyoshi) [inline saleor/graphql/page/sorters.py:24]: Does that make sense to introduce a new filter and instantly deprecate it? Why not only introduce the non-deprecated one?

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]

File: saleor/graphql/schema.graphql (hunk 1 above)
Comment (zedzior) [reply saleor/graphql/schema.graphql:10024]: Going to be fixed in separate PR (https://github.com/saleor/saleor/issues/10968)

**Review by fowczarek — APPROVED** [review]

**Pull request merged**